test/kubernetes_test.go

```diff
@@ -479,8 +827,65 @@ func TestKubernetesIntegrationFallthrough(t *testing.T) {
 	drop 0
     }
 `
-	cases := append(dnsTestCases, dnsTestCasesFallthrough...)
-	doIntegrationTests(t, corefile, cases)
+	doIntegrationTests(t, corefile, dnsTestCasesFallthrough)
+}
+
+//headlessAResponse returns the answer to an A request for the specific name and namespace
```

Comment:
Thanks for adding comments. You don't need to formally describe the input params this should be obvious from the function signature. Function comments should read like a short story. I.e. `godoc .` in this dir should be a "nice read"